aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -172,7 +177,14 @@ void initialize(
       LockManager lock,
       Map<String, String> catalogProps) {
     this.catalogProperties = catalogProps;
-    initialize(name, path, properties, s3Properties, client, lock);
+    initialize(
+        name,
+        path,
+        properties,
+        s3Properties,
+        client,
+        lock,
+        CatalogProperties.UNIQUE_TABLE_LOCATION_DEFAULT);
```

Comment:
why would we always want to pass false here? Wouldn't we want to check what was set in the properties and only default to false if the property wasn't set?